huxley/www/static/js/huxley/components/AdvisorAssignmentsView.js

```diff
@@ -83,6 +131,7 @@ var AdvisorAssignmentsView = React.createClass({
           <td>{committees[assignment.committee].name}</td>
           <td>{countries[assignment.country].name}</td>
           <td>{committees[assignment.committee].delegation_size}</td>
+          <td><input name={assignment.id} type="checkbox"/></td>
```

Comment:
Currently these are uncontrolled components. Instead, they should be controlled components, so that their values can be encoded in state. Use ReactLink to do this: http://facebook.github.io/react/docs/two-way-binding-helpers.html

@tdowds has used it (relatively) extensively and should be able to help you.